backend/Generator/main.py

```
@@ -373,6 +389,22 @@ def process_file(self, file):
             content = self.extract_text_from_pdf(file_path)
         elif file.filename.endswith('.docx'):
             content = self.extract_text_from_docx(file_path)
+        elif file.filename.endswith('.pptx') or file.filename.endswith('.pt'):
+            content = self.extract_text_from_pptx(file_path)
+            if not content:
+                try:
+                    with open(file_path, 'r', encoding='utf-8', errors='ignore') as f:
+                        content = f.read()
+                except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)